### PR TITLE
ipbt: fix the build on < 10.7

### PR DESCRIPTION
**Pull request by barracuda156**
#### Description

Fixes: https://trac.macports.org/ticket/66737

###### Type(s)

- [x] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.6.8 Server
Xcode 3.2.6

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: sysutils/ipbt/Portfile [hunk 1]
```diff
@@ -33,6 +38,9 @@ depends_lib         port:ncurses
 
 cmake.build_type    Debug
 
+# https://trac.macports.org/ticket/66737
+compiler.blacklist-append *gcc-4.* {clang < 400}
```
Comment (jmroot) [inline sysutils/ipbt/Portfile:42]: Why not use `compiler.c_standard`?

Comment (barracuda156) [reply sysutils/ipbt/Portfile:42]: @jmroot It does not solve the issue: https://trac.macports.org/ticket/66737#comment:2

Comment (jmroot) [reply sysutils/ipbt/Portfile:42]: I'm not suggesting using `configure.cflags-append -std=c99`, but rather `compiler.c_standard 2011`.

Comment (barracuda156) [reply sysutils/ipbt/Portfile:42]: Does it require C++11 anywhere? I could have missed that.

Comment (jmroot) [reply sysutils/ipbt/Portfile:42]: It doesn't require C++ at all as far as I can see.

Comment (barracuda156) [reply sysutils/ipbt/Portfile:42]: > It doesn't require C++ at all as far as I can see.

Sorry, I meant C11, of course.

Comment (jmroot) [reply sysutils/ipbt/Portfile:42]: As I mentioned in the ticket, it is using a C11 feature, yes.

Comment (barracuda156) [reply sysutils/ipbt/Portfile:42]: Should I make a PR to change blacklist to c_standard?

**Pull request merged**